Sources/SmileID/Classes/DocumentVerification/Model/DocumentCaptureModel.swift

```diff
@@ -117,6 +126,7 @@ class DocumentCaptureViewModel: ObservableObject, JobSubmittable, ConfirmationDi
                     self.processRectangle(rectangle: rect, imageSize: imageSize)
                 }
             })
+        self.startManualCaptureTimer()
```

Comment:
Move this to `init`, since we are not waiting for any action to complete before starting the timer